src/main/java/uk/gov/companieshouse/confirmationstatementapi/checks/CompanyPscCheck.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.companieshouse.confirmationstatementapi.checks;
+
+import uk.gov.companieshouse.confirmationstatementapi.ConfirmationStatementApiApplication;
+import uk.gov.companieshouse.confirmationstatementapi.exception.ServiceException;
+import uk.gov.companieshouse.confirmationstatementapi.service.PscService;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+public class CompanyPscCheck {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ConfirmationStatementApiApplication.APP_NAME);
+    private final PscService pscService;
+
+    public CompanyPscCheck(PscService pscService) {
+        this.pscService = pscService;
+    }
+
+    public boolean hasMultipleActivePscs(String companyNumber) throws ServiceException {
+        try {
+            var count = pscService.getPscs(companyNumber).getActiveCount();
+            return count != null && count > 1;
+        } catch (ServiceException e) {
+            LOGGER.error(e);
+            throw new ServiceException("Error Retrieving Persons of significant control", e);
```

Comment:
I wouldn't bother catching the exception here just to log and throw it again.
Just let the method throw the exception as its already logged in psc service.
And the controller will handle it when its thrown out of this method